src/browser-pool/pool.ts

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+import Browser from "../browser/browser";
+
+interface Pool<T extends Browser = Browser> {
+    getBrowser(id: string, opts?: object): Promise<T>;
+    freeBrowser(browser: T, opts?: object): Promise<void>;
+    cancel(): void;
+}
+
+export default Pool;
```

Comment:
I'd avoid `export default` absolutely everywhere (at least in new files): https://t.me/why_not_export_default

This file may be named `types.ts` and contain named exports.